Dockerfile

```diff
@@ -103,6 +103,7 @@ RUN microdnf update -y \
     && microdnf install ca-certificates libcap shadow-utils -y \
     && rm -rf /var/cache/yum \
     && chmod -R 777 . \
+    && cp /work/logdna-agent /work/logdna-agent-no-cap \
```

Comment:
This is a neat way to do it, does it increase the size of the binary much? Just wondering if it's worth creating a really tiny shim binary that we can add the setcap bit to which just forks off the agent

Reply:
it does increase image size. thought about wrapper too. it may break existing customer setups (remote debugging or custom yamls referring to agent exe). the better way is to request caps using `libcap` inside agent itself and add ability to disable that in command line. will do it in another pr after this one settles.